### ISSUE TITLE
Only works in root component not in sub component

### ISSUE THREAD

**Issue by j0wey** [foxitsoftware/ngx-foxitpdfsdkforweb#2]
The setup as it is described in the readme doesn't work for child components.

The result is an 404 error for all addons, because they get accessed via the relative path declared in `foxitWebPDFComponent.ts` :  

```
http://localhost:4200/url-to/child-component/node_modules/ngx-foxitpdfsdkforweb/lib/uix-addons/file-property/addon.info.json 
```

**Expected Behaviour:**  

```
{
    path: 'document',
    children: [ 
      {
        path: ':id',
        component: DocumentViewerComponent,
      },
    ],
}
```

For a route like the above, I would expect that it is possible to use `foxit-webpdf` in the `DocumentViewerComponent`.

How could this be sovled?

**Update:**
I created a [sample project](https://github.com/j0wey/ngx-foxit-child-issue) to clarify the problem.
Just `npm install`, `ng serve` and click on the link to the child component.
That's all to reproduce the problem.

**Comment by j0wey** [comment]
Since there seems to be no feedback on issues here...

For anyone who might stumble over this in the future, it is fixed as of version 7.2.0.

**Comment by revl94** [comment]
Hello, i have the same issue, How could this be sovled?... please help, i need use this tool
